sources/packages/backend/apps/api/src/route-controllers/application/models/application.dto.ts

```diff
@@ -161,7 +161,6 @@ export class ApplicationDataChangeAPIOutDTO {
 
 export class ApplicationSupplementalDataAPIOutDTO extends ApplicationBaseAPIOutDTO {
   studentFullName: string;
-  applicationOfferingIntensity?: OfferingIntensity;
```

Comment:
Why is the property removed, the comment is to type as not nullable `applicationOfferingIntensity?: OfferingIntensity;`